Lau/create landing page

Azrux opened the pull request:
## Description
Landing page was added, it includes:
* Main section with background photo and a call to action button to visit Home page
* About Athenas section with 3 collapsible sections for:
  * Why choose us?
  * Testimonials
  * Our products
* Form section to sign up (not working)
* Contact us section with a map

## How to test?
* Run npm i
* Run npm run dev
* Check design and responsive design
* Check that every functionality is working fine

Azrux committed:
feat: create about-athenas component
feat: added component to collapse descriptions
feat: added component to map sections

Azrux committed:
feat: added sign-up form component

Azrux committed:
fix: changes on components
 refactor: add NextUI components to form
 style: change style conflicts on AboutAthenas section

Azrux committed:
feat: added landing
chore: install leaflet library to add geolocalization

Azrux committed:
chore: remove unnecesary comment and add ignore for biome rule

Azrux committed:
chore: remove camel case on section titles

Azrux committed:
Merge branch 'main' into lau/create-landing-page

Azrux committed:
chore: apply format

Azrux committed:
Merge branch 'main' into lau/create-landing-page

Azrux committed:
Merge branch 'main' into lau/create-landing-page

Azrux committed:
Merge branch 'main' into lau/create-landing-page

wellnahuel commented on src/components/about-athenas/info-card/info-card.tsx:
Remove the mb-4 in every screen, mobile,tablet and desktop

![image](https://github.com/user-attachments/assets/fbc26f87-9d7b-4344-bdc7-500331bc1e55)

wellnahuel commented:
Do you want to add a bold to the name of each contact information?

![image](https://github.com/user-attachments/assets/37dcd347-47d4-400e-a256-1ce797459ca0)

wellnahuel commented:
Can you change the bg color of the input? I think that with white there is a better result with the contrasts

![image](https://github.com/user-attachments/assets/affd5e9d-963b-44a6-a790-24308dccdcfa)

wellnahuel commented:
![image](https://github.com/user-attachments/assets/84031c48-8854-4b08-b0c3-4764042ceb76)

With 400px the screen eats my text

wellnahuel commented:
Make these changes and LGTM!

Azrux committed:
fix: resolve comments
fix: fixed styles on collapsibleSection component
style: removed unnecesary class on InfoCard
chore: changed "Únete" with "Unite" in the form section
style: changed background color in the Input component
refactor: changed contact section content

wellnahuel reviewed (changes requested):
Make the changes requested in comments and LGTM!

wellnahuel reviewed (approved):
LGTM

Azrux committed:
fix: fix workflow

Azrux committed:
chore: update lock

Azrux committed:
Merge branch 'main' into lau/create-landing-page
